### PR TITLE
Made Changes

### PR DESCRIPTION
**Pull request by manishnetsol**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -10,6 +10,10 @@
 <?php
 include 'includes/header.php';
 ?>
+
```
Comment (manishnetsol) [inline index.php:13]: @pulkitnagpal007 Please add the div over here.
@dheeth 
@sajalbansal2298 
@mahakbansal3

File: index.php [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
+  <!-- content -->
+
+
```
Comment (pulkitnagpal007) [inline index.php:16]: @manishnetsol This is Test
@dheeth This is Test

**Review by pulkitnagpal007 — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (pulkitnagpal007) [reply index.php:13]: Done Sir.

File: index.php (hunk 2 above)
Comment (manishnetsol) [reply index.php:16]: @pulkitnagpal007 This is done, please review again.

**Review by pulkitnagpal007 — APPROVED** [review]

**Pull request merged**